Common Collections/Strings/Loop Over a Vector/task-info.yaml

```diff
@@ -0,0 +1,15 @@
+type: edu
+files:
+- name: src/main.rs
+  visible: true
+- name: Cargo.toml
+  visible: true
+- name: tests/tests.rs
+  visible: false
+- name: src/lib.rs
+  visible: true
+  placeholders:
+  - offset: 83
+    length: 8
+    placeholder_text: Fill this up so that each element in the Vec `v` is multiplied
```

Comment:
Let's add //at the beginning of the placeholder text, so it won't be painted red by the Idea at the beginning of the exercise